tests/ncdm.lua

```diff
@@ -46,13 +47,13 @@ np_alloc_factor= 4.0      -- Amount of memory allocated for particle
 
 remove_cosmic_variance = true
 
-dealiasing_type = "gaussian"
+dealiasing_type = "none"   --aggressive means GADGET now
```

Comment:
Let's rename `dealiasing_type` to `force_softening_type`, and rename aggressive to 'gadget-long-range' to avoid double-speak.

Reply:
Should I change all the FASTPM_DEALIASING_... to FASTPM_SOFTENING_...?
Also should I change FASTPM_DEALIASING_AGGRESSIVE_GAUSSIAN to FASTPM_DEALIASING_GADGET_LONG_RANGE?